### DISCUSSION TITLE
User profile's pronouns

### DISCUSSION THREAD

**Discussion by MariaPinguelli** [community/community#142315] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello there,
I'm doing research for my final paper, and was looking for a way to get user's pronouns using GitHub's API endpoints , but did not find information about it in the documentation.

Does someone knows if there is a way to get this data?

**Comment by ghost** [comment]
I'm pretty sure the only way is though the user's profile. They have to have selected the pronouns for it to show though.

**Reply by MariaPinguelli** [reply]
Yes, I found it, thank you ❤️

I think I wasn't very clear with my question, I want to get this data using the API endpoint. Since I didn't find anything in the API docs I guessed I just wasn't finding it. (I'm add this to my question too)

**Comment by NazmuzShaad** [comment]
> ### Select Topic Area
> Question
> 
> ### Body
> Hello there, I'm doing research for my final paper, and was looking for a way to get user's pronouns, but did not find information about it in the documentation.
> 
> Does someone knows if there is a way to get this data?

Click on Edit Profile. You will find pronoun options. Select from there and save it.

**Reply by MariaPinguelli** [reply]
Do you know if there's a way to get it from many users using GitHub's API?

**Comment by ebndev** [comment]
Hey @MariaPinguelli! There's some docs mentioned in [this discussion](https://github.com/orgs/community/discussions/142186) that might be helpful for you!

What's your research on, if you don't mind sharing? 😄

**Reply by MariaPinguelli** [reply]
Thanks for the docs it helped a lot!

Sadly, I'm looking for a way to get it from GitHub's API endpoint, do you know if there's a way to do it?

And I don't mind telling about it.
I'm researching metrics that can measure one's feeling of belonging in open source projects.
I want the pronouns to measure how diverse a open source project's community is.

**Reply by ebndev** [reply, marked as answer]
@MariaPinguelli That's so cool you're working on that research! 

Hmm I'm not seeing anything in the [API docs](https://docs.github.com/en/rest/users?apiVersion=2022-11-28) about user's pronouns but you can double check in case I'm missing it. If the feature is not available, please feel free to create a [new discussion](https://github.com/orgs/community/discussions/new?category=api-and-webhooks) in our `API & Webhooks` category to leave product feedback!

**Reply by MariaPinguelli** [reply]
Thanks <3

I will do what you suggested :)

**Comment by norwd** [comment]
If this ever becomes available in the API, maybe @github could look into using this in the automatically generated release notes?

Currently, when a user makes their first contribution to a repository, when that contribution appears in a release, the default pronoun is `their`:

![image](https://github.com/user-attachments/assets/f08d0a24-0d0c-4a26-a723-6a898c3b3dc8)

But now that GitHub users can set well-known pronouns from a drop-down, this could automatically use those, e.g. in this edited example:

![image](https://github.com/user-attachments/assets/e458bc74-3efd-4ee0-9259-a1d3a15a69c0)
